src/KeyringController.ts

```diff
@@ -1036,7 +1036,9 @@ class KeyringController extends EventEmitter {
 
     await keyring.deserialize(data);
 
+    // @ts-expect-error The method 'init' is not part of the current Keyring type
     if (keyring.init) {
+      // @ts-expect-error The method 'init' is not part of the current Keyring type
```

Comment:
Nit: We can check for this function in a way that TypeScript understands, rather than suppress the error:

```suggestion
    if (hasProperty(keyring, 'init') && typeof keyring.init === 'function') {
```

The `hasProperty` function used here would have to be imported, but it's in `@metamask/utils` which is already a dependency.

Reply:
I'll open another PR to include your suggestion @Gudahtt